biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceUtils.java

```diff
@@ -552,4 +552,10 @@ public static VersionedClause toVersionClause(Resource resource, String mask) {
 		return new VersionedClause(identity, attribs);
 	}
 
+	static <T> T requireNonNull(T obj) {
```

Comment:
I thought about putting it in aQute.lib somewhere, but it didn't seem worth it for a simple method to drag in another class/package. When we move to Java 8, we can remove this and replace with using Objects.